src/listener/githubIssueReference.ts

```diff
@@ -0,0 +1,75 @@
+import { Listener } from 'discord-akairo';
+import { Message, MessageEmbed } from 'discord.js';
+import { truncate } from 'lodash';
+import { guildConfigs } from '../guild/config/guildConfigs';
+import { capitalize } from 'lodash';
+import { MinehutClient } from '../client/minehutClient';
+import { getIssue } from '../util/functions';
+
+export default class GithubIssueReferenceListener extends Listener {
+	client: MinehutClient;
+
+	constructor(client: MinehutClient) {
+		super('githubIssueReference', {
+			emitter: 'commandHandler',
+			event: 'messageInvalid',
+		});
+		this.client = client;
+	}
+	async exec(msg: Message) {
+		if (!msg.guild) return;
+		const config = guildConfigs.get(msg.guild.id);
+		if (!config || !config.features.githubIssue) return;
+		if (config.features.githubIssue.ignoredChannels?.includes(msg.channel.id))
+			return;
+		const messageMatch = msg.content.match(/#\d+/);
+
+		if (messageMatch) {
+			const issueNumber = parseInt(messageMatch[0].slice(1));
+			if (
+				this.client.githubCooldownManager.isOnCooldown(
+					`gh-${issueNumber}-${msg.channel.id}`
+				)
+			)
+				return msg.react('⏲️');
```

Comment:
Check the spacing here

Reply:
Weird, prettier likes it indented there, 🤷